gio/src/auto/file.rs

```diff
@@ -56,7 +57,8 @@ impl File {
     }
 
     #[doc(alias = "g_file_new_for_commandline_arg_and_cwd")]
-    pub fn new_for_commandline_arg_and_cwd<P: AsRef<std::ffi::OsStr>, Q: AsRef<std::path::Path>>(
```

Comment:
Should maybe autodetect `new_for` like we do `new_from`?

Reply:
We already do https://github.com/gtk-rs/gir/blob/b5ff6f8dd8713c3a8d4864f2f975374f0612976b/src/analysis/functions.rs#L558

But that applies only to constructors.

Reply:
Oh I see!